apps/interpolate_sample.py

```diff
@@ -116,21 +148,38 @@ def sample_from_path(generator,
                      latent_a,
                      latent_b,
                      intervals,
+                     label_a=None,
+                     label_b=None,
                      interp_mode='lerp',
                      space='z',
                      **kwargs):
-    interp_alphas = np.linspace(0, 1, intervals)
+    interp_alphas = torch.linspace(0, 1, intervals)
     interp_samples = []
 
+    if label_a is not None and label_b is not None:
+        device = get_module_device(generator)
+        embedding_a = generator.shared_embedding(label_a.to(device))
```

Comment:
May different models have different names for embedding layers?
Seems `shared_embedding` is only suitable for the BigGAN family?

Reply:
True. I have two ideas, one is to set a unified name for all embedding layers, the other is to set a script argument for users to transfer the embedding layer's name. What's your opinion?

Reply:
Script argument may be better?
And we may also pre-define a `dict` for `model-embedding` name mapping?

e.g., `embedding_mapping = dict(BigGAN='shared_embedding')`

Then name of embedding can be accessed by

```[language=python]
if embedding_name is None:
    assert model_name in embedding_mapping, 'SOME INFO'
    embedding_name = embedding_mapping[model_name]

embedding_fn = getattr(generator, embedding_name)
embedding_a, embedding_b = embedding_fn(label_a.to(device)), embedding_fn(label_b.to(device))
```

Reply:
Some biggan models may not have `shared_embedding`? How do you deal with the models with a split embedding?

Reply:
BigGAN models always have a `shared_embedding` layer. If a model has a split embedding,  `shared_embedding`  would be an `Identity` layer, and this case has also be tested.